src/components/protocol_handler/test/incoming_data_handler_test.cc

```diff
@@ -56,6 +56,8 @@ class IncomingDataHandlerTest : public ::testing::Test {
     some_data2_size = 512;
     some_data = new uint8_t[some_data_size];
     some_data2 = new uint8_t[some_data2_size];
+    memset(some_data, 0xAB, some_data_size);
+    memset(some_data2, 0xAB, some_data2_size);
```

Comment:
@vlantonov memset() is evil. Why not to use something like 
`unsigned int p6[10] = {89} ;  // will set the 10 values to 89`